introduction.md

```diff
@@ -1,2 +1,7 @@
 Hi my name is Chi.
-My favourite codeschool path is Electives.
\ No newline at end of file
+
+My favourite codeschool path is Electives.
+
+* WordPress
+* PHP
+* Github
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?